### PR TITLE
portsyncd: Adding sample folder with interfaces/vlan_interfaces file

### PR DESCRIPTION
**Pull request by stcheng**
Addressing https://github.com/Azure/sonic-swss/issues/47

### CODE REVIEW

File: portsyncd/sample/interfaces [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# interfaces file
+#
+# The main configuration file that is located in folder /etc/network/.
+
+auto Ethernet0
+iface Ethernet0 inet static
+    address 10.0.0.0
+    netmask 255.255.255.254
```
Comment (lguohan) [inline portsyncd/sample/interfaces]: can you add Ethernet4 in this example? as it is referenced in the vlan_interfaces.

**Comment by stcheng** [comment]
updated.

File: portsyncd/sample-dell-s6000/interfaces [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+auto eth0
+iface eth0 inet static
+    address 10.3.147.1
+    netmask 255.255.254.0
+    up ip route add default via 10.3.146.1 dev eth0 table default
+    up ip route add default via 10.3.146.1 dev eth0 table 1
+    up ip rule add from 10.3.147.171/32 table 1
+    down ip route delete default via 10.3.146.1 dev eth0 table default
+    down ip route delete default via 10.3.146.1 dev eth0 table 1
+    down ip rule delete from 10.3.147.171/32 table 1
```
Comment (lguohan) [inline portsyncd/sample-dell-s6000/interfaces]: remove all the ip route/ip rule

replace with gateway 10.3.146.1

Comment (stcheng) [inline portsyncd/sample-dell-s6000/interfaces]: resolved.

**Comment by lguohan** [comment]
Ack

File: portsyncd/sample-dell-s6000/vlan_interfaces [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+#
+# By default, this file shall be put in folder /etc/network/interfaces.d/.
+
+# The following configurations configures a VLAN with VLAN ID 1200, which
+# contains two VLAN members Ethernet0 and Ethernet4.
+auto Vlan1200
+iface Vlan1200 inet static
+    address 20.0.0.0
+    netmask 255.255.192.0
+    bridge_ports Ethernet0 Ethernet4
```
Comment (dplore) [inline portsyncd/sample-dell-s6000/vlan_interfaces]: Hi,  Is this intended to be compatible with the  portsyncd/sample-dell-s6000/interfaces  file?  Ethernet0 and Ethernet4 have IP addresses assigned also.  What is the intended behavior?  Perhaps we should show an example which includes the requirements for both the interfaces file and the vlan_interfaces file when vlan's exist.

**Pull request merged**